src/main/java/sorald/FileUtils.java

```diff
@@ -16,14 +16,21 @@
 public class FileUtils {
 
     /**
-     * Compare the two given paths as absolute, normalized paths.
+     * Compare the two given paths as real paths, resolving any symbolic links.
+     *
+     * <p>Note that this method returns false if either path does not exist, even if the paths are
+     * otherwise equal!
      *
      * @param lhs A path.
      * @param rhs A path.
-     * @return Whether or not the paths are equal as absolute, normalized paths.
+     * @return true iff both paths exist and point to the same real file
```

Comment:
Double f in "if".

Reply:
That's intentional, it's stronger than "if": https://en.wikipedia.org/wiki/If_and_only_if

Perhaps this isn't as commonly accepted as KTH's math courses lead you to believe? Change it to "if"?

Reply:
> it's stronger than "if": https://en.wikipedia.org/wiki/If_and_only_if

Oh, I didn't know about that. Let's keep it like it is then!

Reply:
> Oh, I didn't know about that. Let's keep it like it is then!

I mean, if you didn't know about "iff", that's a bit of a red flag in terms of using it :D

Then again, to anyone who hasn't seen "iff" before it will just look like a typo and so it doesn't really matter here. But in general, if you or anyone else reviewing a PR doesn't understand a term, then it's likely it's not the best term to use!